compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1260,6 +1260,10 @@ rustc_queries! {
             eval_always
             desc { "looking up the paths for extern crates" }
         }
+
+        query needs_gdb_debug_scripts_section(_: CrateNum) -> bool {
+            desc { "determine if the crate needs a gdb debug scripts section" }
```

Comment:
This only works for the local crate, right?
```suggestion
            desc { "determine if the current crate needs a gdb debug scripts section" }
```